tests/test_integration/test_terminal_bench_v1.py

```
@@ -244,3 +250,74 @@ def test_filters_to_known_tasks(self, mock_ensure, tmp_path):
+        assert TerminalBenchHardAASplit._subset_category("make-doom-for-mips") == "unsolvable"
+        assert TerminalBenchHardAASplit._subset_category("run-pdp11-code") == "unsolvable"
+        assert TerminalBenchHardAASplit._subset_category("aimo-airline-departures") == "solvable"
+        assert TerminalBenchHardAASplit._subset_category("play-zork") == "solvable"
+
+    def test_base_benchmark_leaves_category_unset(self):
+        # The 47-task variant declares no unsolvable set, so it must not tag a
+        # category (preserves existing behavior; no spurious breakdown).
+        assert TerminalBenchHard._subset_category("make-doom-for-mips") is None
+
+    async def test_seed_tags_category(self, monkeypatch):
+        async def fake_seed(self, idx):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `fake_seed`

(ANN202)